Assets/MRTK/Examples/Demos/HandTracking/Profiles/HandMenuExamplesConfigurationProfile.asset

```diff
@@ -0,0 +1,55 @@
+%YAML 1.1
```

Comment:
Do we need this new profile?

Reply:
Yes! By default eye gaze isn't enabled in the input simulator, meaning we have to go in there to change how we simulate that in the editor.

Reply:
Ah, that brings up an interesting question of what should happen if eye gaze is not availabe for some reason -- should the behavior default to the head gaze in that case?

Reply:
Yup! It actually resolves to that due to the way that I cast the GazeProvider (which does both eye and head gaze)

Reply:
I don't know if Eye gaze should be used at all here. @Kjakubzak is there a case where you found the headgaze to be insufficient for this? When working with menus like this, users will dart their eyes around, and it could cause some sort of flickering if your tolerances are too narrow.

Reply:
The nice thing about this particular implementation is that it works with both head gaze and eye gaze. I'd prefer to keep eye gaze as a component of it since as AR/VR progresses, there will be more hardware that supports eye gaze (meaning it will become more and more present in experiences). Displaying utilizations for eye gaze now (that create intuitive interactions) is important so developers and designers can have a great place to start so they can explore the possibilities of the tech.

That aside, the code itself only uses eye-gaze/gaze initially to confirm the menu triggering/activating. It doesn't require continuous eye-gaze/gaze for the menu to remain attached to the hand.

As an aside, In general, I find using headgaze rather than eyegaze can potentially introduce more cognitive load since it requires a larger set of intentional movements to trigger interactions (HL1 gaze + tap for me was excruciating in comparison to eyegaze + commit)

Reply:
@Kjakubzak since you added this new set of profiles only to allow input simulation of eye gaze, but you also coded it so that it will use head gaze as a fallback if eye gaze isn't available, could you actually just remove all these profiles? It seems like the code will still fall back to head gaze, and in the simulator the same thing will happen as if you had eye gaze in the input simulation profile.

Reply:
Removed!